api/src/main/java/org/apache/cloudstack/api/response/NetworkOfferingResponse.java

```diff
@@ -99,6 +99,10 @@
     @Param(description = "true if network offering can be used by VPC networks only")
     private Boolean forVpc;
 
+    @SerializedName(ApiConstants.FOR_TUNGSTEN)
+    @Param(description = "true if network offering can be used by tungsten networks only")
```

Comment:
Another example of "tungsten" used in end user docs vs "tungsten fabric"

Reply:
```suggestion
    @Param(description = "true if network offering can be used by Tungsten-Fabric networks only")
```